components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -456,6 +500,9 @@
 <config_btr_dt ocn_grid="WCAtl12to45E2r4">'0000_00:00:15'</config_btr_dt>
 <config_btr_dt ocn_grid="SOwISC12to60E2r4">'0000_00:00:15'</config_btr_dt>
 <config_btr_dt ocn_grid="ECwISC30to60E2r1">'0000_00:01:00'</config_btr_dt>
+<config_btr_dt ocn_grid="ECwISC30to60E3r1">'0000_00:01:00'</config_btr_dt>
+<config_btr_dt ocn_grid="IcoswISC30E3r1">'0000_00:01:00'</config_btr_dt>
```

Comment:
```suggestion
<config_btr_dt ocn_grid="IcoswISC30E3r1">'0000_00:00:45'</config_btr_dt>
```
Even with a Haney number of 10, I wasn't able to get a stable `simulation` run with 1 minute `btr_dt` so I think this is going to have to be 45 seconds unless more spin-up allows us to relax this.